frontend/app/src/components/history/events/tx/RepullingTransactionFormDialog.vue

```diff
@@ -14,18 +15,16 @@ import { TaskType } from '@/types/task-type';
 import { logger } from '@/utils/logging';
 
 const modelValue = defineModel<boolean>({ required: true });
+const currentAction = defineModel<HistoryEventAction>('currentAction', { required: true });
 
-withDefaults(defineProps<{
+const props = withDefaults(defineProps<{
+  eventHandlers?: DialogEventHandlers;
```

Comment:
Can't we still do e discriminating union and a single handler on the consumer.
I would prefer it than passing the event handlers directly to the child component.

Reply:
so the emit won't be executed. I'm not sure about the discriminating union thing.
But now I change it so we only pass the 2 necessary function, not the whole event handlers.